ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -870,6 +870,21 @@ AHU_DFSS_DXZC_ECOND_ZHM:
   - ECOND
   - ZHM
 
+AHU_SS_CHWDT_STM_RTM_RWISOVPC:
```

Comment:
incorrect use of SS (should be SFSS) and mismatching of terms like discharge and supply; no mention of outisde air temperature control, requisite for modeling any AHU. you also use an optional return air temperature setpoint, which should exist as a required field on an abstract type.

please reconstruct.

Reply:
Updated the model